src/actions/LogActions.tsx

```diff
@@ -16,6 +16,13 @@ import { sendLogs } from '../modules/Logs/api'
 import { ThunkAction } from '../types/reduxTypes'
 import { clearLogs, logWithType, readLogs } from '../util/logger'
 
+const SENSITIVE_KEY_REGEX = /(allKeys|displayPrivateSeed|displayPublicSeed|otpKey|loginKey|recoveryKey|dataKey|syncKey)/gm
```

Comment:
You don't need the parentheses or the 'm' flag. They do nothing in this particular regexp. So just: `/allKeys|displayPrivateSeed|displayPublicSeed|otpKey|loginKey|recoveryKey|dataKey|syncKey/g`